docs_source_files/content/webdriver/page_loading_strategy.de.md

```diff
@@ -9,6 +9,17 @@ it follows the _normal_ pageLoadStrategy.
 It is always recommended to stop downloading additional 
 resources (like images, css, js) when the page loading takes lot of time.
 
+`document.readyState` property of a document describes the loading state of current document.
```

Comment:
```suggestion
The `document.readyState` property describes the loading state of the current document.
```

Reply:
Could you please have a look at this suggestion @Harsha509?

Reply:
Apologies @diemol. Updated just now